apiserver/admin_test.go

```diff
@@ -804,13 +787,49 @@ func (s *loginSuite) TestOtherEnvironmentWhenNotController(c *gc.C) {
 	defer envState.Close()
 	info.ModelTag = envState.ModelTag()
 	st := s.openAPIWithoutLogin(c, info)
-	defer st.Close()
 
 	err := st.Login(machine.Tag(), password, "nonce", nil)
-	c.Assert(errors.Cause(err), gc.DeepEquals, &rpc.RequestError{
-		Message: "permission denied",
-		Code:    "unauthorized access",
+	assertPermissionDenied(c, err)
+}
+
+func (s *loginSuite) loginUserWithAccess(c *gc.C, info *api.Info) (*state.User, params.LoginResult) {
```

Comment:
what does "WithAccess" mean here?
It seems like it should mean "log in this specified user with this specified access" but that doesn't seem to be the case?